node/src/Commands.ts

```diff
@@ -494,6 +494,300 @@ export function createSetBit(
     ]);
 }
 
+/**
+ * Represents a signed or unsigned argument encoding for the {@link BaseClient.bitfield|bitfield} or
+ * {@link BaseClient.bitfieldReadOnly|bitfieldReadOnly} commands.
+ */
+export interface BitEncoding {
+    /**
+     * Returns the encoding as a string argument to be used in the {@link BaseClient.bitfield|bitfield} or
+     * {@link BaseClient.bitfieldReadOnly|bitfieldReadOnly} commands.
+     *
+     * @returns The encoding as a string argument.
+     */
+    toArg(): string;
```

Comment:
this breaks the convention established to include the options logic in the createX function.  Then we can keep these as types only, and we don't need to inherit any interfaces.

Reply:
In essence, we can delete this interface and move all of the toArg logic to `createBitField`

Reply:
As discussed in standup, it sounds like we'll stick with classes for these 2 commands for now